### PR TITLE
template/koa-rest-api: `koa-bodyparser` => `@koa/bodyparser`

### PR DESCRIPTION
**Pull request by 72636c**
https://github.com/seek-oss/skuba/pull/1603#pullrequestreview-2152596387

### CODE REVIEW

**Comment by 72636c** [comment]
Do we want a `patch` for this?

**Review by AaronMoat — APPROVED** [review]

**Comment by AaronMoat** [comment]
I'd hold off on the patch until https://github.com/koajs/bodyparser/issues/161 settles, personally

**Comment by samchungy** [comment]
Should we renovate replace it?

File: template/koa-rest-api/package.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
   },
   "devDependencies": {
     "@types/chance": "^1.1.3",
+    "@types/co-body": "^6.1.3",
```
Comment (samchungy) [inline template/koa-rest-api/package.json:36]: what's this

**Comment by AaronMoat** [comment]
🤔 I think `@koa/bodyparser` bundles with types

File: template/koa-rest-api/package.json (hunk 1 above)
Comment (72636c) [reply template/koa-rest-api/package.json:36]: To  un`skipLibCheck`, we need to pull in the transitive type dep 🥲

https://github.com/koajs/bodyparser/blob/031348a2d469dd288dc21cd10d8de280d9936f2c/package.json#L49

https://github.com/koajs/bodyparser/blob/031348a2d469dd288dc21cd10d8de280d9936f2c/src/body-parser.types.ts#L1

Comment (AaronMoat) [reply template/koa-rest-api/package.json:36]: 😢 worth upstreaming?

**Review by tadhglewis — APPROVED** [review]

File: template/koa-rest-api/package.json (hunk 1 above)
Comment (samchungy) [reply template/koa-rest-api/package.json:36]: https://github.com/koajs/bodyparser/pull/164

Comment (72636c) [reply template/koa-rest-api/package.json:36]: Thanks @samchungy 🙇

**Pull request merged**